jingbao/1939/04/jb_3797_1939-04-23_0006+0007.xml

```diff
@@ -1043,7 +1043,7 @@
                         姿秀逸，應酬周到，顧客得識此姝，<lb/>
                         洵有樂不思蜀之感，該社電話，爲九<lb/>
                         〇八六六…九六六八四號，謂予不信<lb/>
-                        招來一試，方信言之不謬也。<lb/>
+			      　<!-- There is a space in this version, and it shows a comma on other day.-->招來一試，方信言之不謬也。<lb/>
```

Comment:
in this case just insert a space (full width "ideographic space"). the comment would be of more help if you'd give an example for the text WITH comma instead of space. i.e. provide a specific date and page where the variant can be found.

Reply:
For this case, I have already inserted a space (full width "ideographic space"). In the same advertisement on 1939.04.25_Page 6, it shows a comma. So, shall I add this supplement to the comment now? Or do it after the "pull request"?